stream-chat-android-compose-sample/src/main/res/values-night/themes.xml

```diff
@@ -2,15 +2,13 @@
     <!-- Base application theme. -->
     <style name="Theme.Streamchatcompose" parent="Theme.MaterialComponents.DayNight.NoActionBar">
         <!-- Primary brand color. -->
-        <item name="colorPrimary">@color/purple_200</item>
-        <item name="colorPrimaryVariant">@color/purple_700</item>
+        <item name="colorPrimary">@color/stream_compose_primary_accent</item>
         <item name="colorOnPrimary">@color/black</item>
         <!-- Secondary brand color. -->
         <item name="colorSecondary">@color/teal_200</item>
-        <item name="colorSecondaryVariant">@color/teal_200</item>
         <item name="colorOnSecondary">@color/black</item>
         <!-- Status bar color. -->
-        <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
+        <item name="android:statusBarColor" tools:targetApi="l">@android:color/transparent</item>
         <!-- Customize your theme here. -->
```

Comment:
Status bar items are now visible now because they are white (in the light theme) and black (in the dark theme). Probably we should add this to the light theme:
```XML
<item name="android:windowLightStatusBar">true</item> 
```
and 
```XML
<item name="android:windowLightStatusBar">false </item> 
```
to the dark theme, to make status bar items tinted properly.

Reply:
BTW, so weird this is not a standard behavior of DayNight theme

Reply:
I figured I'd just follow the default behavior (might need to update UI components too with this).

I'll apply these changes and update the PR!